lib/client.coffee

```diff
@@ -9,29 +20,74 @@ module.exports = class IntuitClient
     [body, done] = [{}, body] if typeof(body) is "function"
     new Request(@options)[method]("#{BASE_URL}#{path}", body, done)
 
-  institutions: (done) ->
-    @request "get", "/institutions", done
+  institutions: (userId, done) ->
+    @options.userId = userId
```

Comment:
It worth making userId a required attribute on the class initializer? not too sure how you get the userId (the example just sets it). This is the company id or individual user ids?